.gitignore

```
@@ -1,6 +1,7 @@
 # Editors
 .idea/*
 !.idea/runConfigurations
+lapis/.idea/libraries
```

Nitpick: We could simply ignore .idea dirs everywhere:
```suggestion
**/.idea/*
!.idea/runConfigurations
```
Or was there a reason why you specifically ignored `lapis/.idea/libraries`?